projects/models.py

```diff
@@ -118,6 +119,7 @@ class Status(models.TextChoices):
         null=False,
         help_text="The Project Control status; completed, incomplete, or not started",
     )
+    remarks = models.TextField(blank=True)
```

Comment:
Just curious: why not "comment"?

Reply:
In OSCAL they are Remarks so I was trying to keep it consistent.